src/modules/vendor/components/courses/CourseCard.vue

```diff
@@ -1,17 +1,54 @@
 <template>
   <q-card flat>
-    <q-card-section class="text-weight-bold text-capitalize" @click="$emit('click', course)">
-      {{ course.name }}
+    <q-card-section @click="$emit('click', course)">
+      <div class="title-text text-weight-bold">{{ course.name }}</div>
+      <div class="items-container">
+        <q-item v-for="info in headerInfo" :key="info.icon" class="item-section-container text-weight-bold">
+          <q-item-section side><q-icon size="12px" :name="info.icon"/></q-item-section>
+          <q-item-section>{{ info.label }}</q-item-section>
+        </q-item>
+      </div>
+      <q-item class="infos-course-container text-weight-bold">
+        <q-item-section><q-icon size="xs" name="people"/> ({{ traineesNbr }}) </q-item-section>
+      </q-item>
+      <q-item class="infos-course-container text-weight-bold">
+        <q-item-section><q-icon size="xs" name="date_range"/> ({{ courseSlots }}) </q-item-section>
+      </q-item>
     </q-card-section>
   </q-card>
 </template>
 
 <script>
+import get from 'lodash/get';
+import groupBy from 'lodash/groupBy';
+import { courseMixin } from '../../mixins/courseMixin';
+import { formatIdentity } from '@helpers/utils';
+
 export default {
   name: 'CourseDetail',
+  mixins: [courseMixin],
   props: {
     course: { type: Object, default: () => ({}) },
   },
+  computed: {
+    trainerName () {
+      return formatIdentity(get(this.course, 'trainer.identity', ''), 'FL');
```

Comment:
get(this.course, 'trainer.identity') || ''

Reply:
tu peux juste envoyer `get(this.course, 'trainer.identity')` car il me semble que la méthode `formatIdentity` prend en charge le cas où l'objet n'existe pas (ou vaut `null`)

Reply:
Done !